src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+p {
+  margin: 0;
+}
+
+.card {
+  height: 406px;
+  width: 198px;
+
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
```

Comment:
According to the checklist, the block should not have fixed dimensions. It's better to use min-width or max-width for width and height should be able to grow based on its content.